paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadataTest.kt

```diff
@@ -727,6 +728,8 @@ internal class PaymentMethodMetadataTest {
         assertThat(metadata.isExternalPaymentMethod("card")).isFalse()
     }
 
+    @OptIn(ExperimentalCardBrandFilteringApi::class)
+    @Suppress("LongMethod", "MaxLineLength")
```

Comment:
We typically suppress these with `./gradlew detektBaseline`. That being said, is there any way to refactor to not need it?

Reply:
Got it, refactored!